core/app/models/spree/order.rb

```diff
@@ -355,7 +355,7 @@ def create_tax_charge!
     def outstanding_balance
       # If reimbursement has happened add it back to total to prevent balance_due payment state
```

Comment:
I think these two comment lines do not apply anymore since we are not adding back any refund (but we are removing it). I'd probably remove them at all since the code is self-explanatory now.